config/CLI/trainer.yaml

```diff
@@ -0,0 +1,49 @@
+trainer:
+  fast_dev_run: False
+  # default_root_dir: "/scratch/shared/py4cast/logs/test_cli"
+  num_nodes: 1
+  devices: "auto"
+  max_epochs: 4
+  deterministic: True
+  strategy: "ddp"
+  accumulate_grad_batches: 10
+  accelerator: "auto"
+  logger: 
+    class_path: lightning.pytorch.loggers.TensorBoardLogger
+    init_args:
+      name: "folder_logger"
+      version: "alpha"
+      default_hp_metric: False
+  profiler: null
+  log_every_n_steps: 1
+  callbacks:
+    - class_path: lightning.pytorch.callbacks.ModelCheckpoint
+      init_args:
+        filename: "{epoch:02d}-{val_mean_loss:.2f}"
+        monitor: "val_mean_loss"
+        mode: "min"
+        save_top_k: 1  # Save only the best model
+        save_last: True  # Also save the last model
+    - class_path: lightning.pytorch.callbacks.LearningRateMonitor
+      init_args:
+        logging_interval: "step"
+    - class_path: lightning.pytorch.callbacks.EarlyStopping
+      init_args:
+        monitor: "val_mean_loss"
+        mode: "min"
+        patience: 50
+    - class_path: py4cast.cli.GribWriter
+      init_args:
+        active: False
+        write_interval: 'epoch'
+        output_dir: "/scratch/shared/py4cast/gribs_writing/test/"
+        template_grib: /scratch/shared/py4cast/gribs_writing/template/grid.arome-forecast.eurw1s40+0001:00.grib
+        output_kwargs: ["poesy_eurw1s40"]
+        sample_identifiers : ["member", "date", "leadtime"]
+        output_fmt: "grid.emul_aro_ai_dataset_{}_member_{}_date_{}_ech_{}.grib"
+
+  check_val_every_n_epoch: 1
+  precision: 32
+  limit_train_batches: 20
+  limit_val_batches: 20
+  limit_test_batches: 20
```

Comment:
I don't think its a good idea to limit the number of batches in the default config file, it should represent a "standard" training experiment. I would remove this. If a user wants to make a quick train to check every thing is alright, they can use the `--trainer.fast_dev_run=True` arguments which limits the training to 1 epoch and 1 batch.

Reply:
yes you are right, I have corrected it